docs/toc.md

```diff
@@ -10,6 +9,33 @@
 ### [🔧 Development using vim](getting-started/devenv/using-vim.md)
 ### [🔧 Development using Sublime Text](getting-started/devenv/using-sublime.md)
 ## [Writing your first console application on .NET Core](getting-started/first-console-app.md)
+# [Scenarios](scenarios/index.md)
+## [Writing console applications](scenarios/console/index.md)
+### [🔧 Overview and moving pieces](scenarios/console/overview.md)
+### [🔧 Console applications paradigm](scenarios/console/paradigm.md)
```

Comment:
I was thinking that there would be very limited conceptual content under "scenarios". All the conceptual content should be under "Concepts". I was thinking that "Scenarios" would exclusively be "Here is how you do topical scenario X + some best practice guidance" and of course link to the concepts section as needed.

For scenarios, I am hoping we eventually grow into more narrow scenarios, like  "Making a Windows Service".